lib/modules/hosts/xkcd.js

```diff
@@ -0,0 +1,24 @@
+addLibrary('mediaHosts', 'xkcd', {
+	domains: ['xkcd.com'],
+	attribution: false,
```

Comment:
You can use something like `logo: 'https://xkcd.com/s/919f27.ico'` instead of `attribution: false` to show the favicon below the image, e.g.

![](http://i.imgur.com/Nd3QSpo.png)